### PR TITLE
Add crowdloan to SafeCallFilter

### PR DESCRIPTION
**Pull request by hqwangningbo**
Hello, members of the parity team.
We are the bifrost team, because of the version upgrade of polkadot-v0.9.38, the addition of SafeCallFilter has caused our SALP protocol to no longer be used. 
The protocol has been running for a long time, and the protocol uses the Transact command to call crowdloan. 
Want to know why crowdloan is not added to the list? 
Hope to add crowdloan to the whitelist.

### CODE REVIEW

File: runtime/kusama/src/xcm_config.rs [hunk 1]
```diff
@@ -172,6 +172,7 @@ impl Contains<RuntimeCall> for SafeCallFilter {
 			RuntimeCall::Timestamp(..) |
 			RuntimeCall::Indices(..) |
 			RuntimeCall::Balances(..) |
+			RuntimeCall::Crowdloan(..) |
```
Comment (KiChjang) [inline runtime/kusama/src/xcm_config.rs:175]: Unfortunately, we can't add every extrinsic from the crowdloan pallet to this list -- we must omit the `add_memo` extrinsic.

```suggestion
			RuntimeCall::Crowdloan(
				crowdloan::Call::create { .. } |
				crowdloan::Call::contribute { .. } |
				crowdloan::Call::withdraw { .. } |
				crowdloan::Call::refund { .. } |
				crowdloan::Call::dissolve { .. } |
				crowdloan::Call::edit { .. } |
				crowdloan::Call::poke { .. } |
				crowdloan::Call::contribute_all { .. }
			) |
```

**Comment by hqwangningbo** [comment]
@KiChjang Thank you for your review, these methods are enough for use now, I have modified the code.

**Comment by hqwangningbo** [comment]
@KiChjang @ggwpez Can you help us with the process of merging?

**Review by ggwpez — APPROVED** [review]

**Comment by hqwangningbo** [comment]
@KiChjang @ggwpez I hope the parity team members can help to deal with it as soon as possible, sorry for the trouble 🙏

**Comment by hqwangningbo** [comment]
@KiChjang I understand, how long does it usually take?

**Review by bkchr — APPROVED** [review]

**Comment by hqwangningbo** [comment]
@ggwpez @KiChjang @bkchr thanks for your work💗💗💗.

**Review by sam0x17 — APPROVED** [review]

**Review by andresilva — APPROVED** [review]

**Comment by ggwpez** [comment]
bot merge

**Pull request merged**